test/e2e/operator.go

```diff
@@ -279,61 +278,6 @@ var _ = Describe("ARO Operator - Conditions", func() {
 	})
 })
 
-var _ = Describe("ARO Operator - MachineSet Controller", func() {
```

Comment:
I agree that the test is adding a lot of time and e2e failures to our release process, but maybe it would be better to disable the test with `XDescribe` and create a WI to improve the test / change the test.

One idea would be a quick test that makes a MachineSet change (something non-destructive) and at least confirms that there was a reconcile. What do you think?

Reply:
I don't think that testing that a machineset scales up and down is that useful for us as an end-to-end test. We already know that the permissions in the cluster are correct (because it scaled up to 3 nodes in the first place) and we don't really need to test that OpenShift is functioning as it should (that should be OCP's job).

Reply:
Makes sense. In that case, I'd probably pull out the old test as well and adjust billing e2e if needed. This test was meant to test our own reconciler but agree we may be relying too much on Azure's ability to make new VMs quickly (or openshift-machine-api).

Reply:
Good idea, I've deleted the old tests as well. I kept the check, since it is perhaps still just good for reasonableness checking.